scripts/gtk-bundle-from-msys2.sh

```diff
@@ -212,6 +212,10 @@ copy_grep_and_dependencies() {
 	if [ "${own_arch}" == "${ABI}" -o -z "${MSYS2_ABI_PATH}" ]; then
 		bin_dir="/usr/bin"
 	else
+		# TODO we could maybe download MSYS2 from http://www2.futureware.at/~nickoe/msys2-mirror/distrib/
+		# extract the relevant files, download "grep" from
+		# http://www2.futureware.at/~nickoe/msys2-mirror/msys/${ABI}/grep-2.XX-1-${ABI}.pkg.tar.xz
+		# and so automate this even more than requiring an existing separate MSYS installation
```

Comment:
I think this comment is weird. The script is meant to be run from an MSYS2 shell.

Besides, you can use pacman to download the archive files, the other parts of the bundle script do that.